source/max-staleness/max-staleness-tests.md

```diff
@@ -0,0 +1,142 @@
+# Max Staleness Tests
+
+These prose tests supplement the machine-readable YAML tests.
+
+## Connection string
+
+Tests of drivers' connection string parsing and validation.
+
+### Basic
+
+maxStalenessSeconds is set to 120 from this connection string:
+
+```
+mongodb://host/?readPreference=secondary&maxStalenessSeconds=120
+```
+
+### No maxStalenessSeconds with mode "primary"
+
+These connection strings are invalid because the mode is "primary", either by default or explicitly:
+
+```
+mongodb://host/?maxStalenessSeconds=120
+mongodb://host/?readPreference=primary&maxStalenessSeconds=120
+```
+
+### Specifying "no max staleness" with "-1"
+
+A MongoClient with this connection string has no max staleness, the same as if "maxStalenessSeconds" were omitted from
+the connection string:
```

Comment:
It looks like we impose a 120 character line limit that is not explicitly defined in the `migrate_to_md.py` script, is this behavior intended?

Reply:
The config is [here](https://github.com/mongodb/specifications/blob/39add29e0a3b6721688845874383878b90811a12/.pre-commit-config.yaml#L41)